cms/templates/widgets/header.html

```diff
@@ -43,7 +43,7 @@ <h2 class="info-course">
         </a>
       </h2>
 
-      <nav class="nav-course nav-dd ui-left">
+      <nav class="nav-course nav-dd ui-left" aria-label="${_('{course_name}').format(course_name=context_course.display_name_with_default)}">
         <h2 class="sr">${_("Navigation for {course_name}").format(course_name=context_course.display_name_with_default)}</h2>
```

Comment:
While you're in this file, you might as well change this value to "Course Navigation" since the course name is not important here.